libs/s25main/Playlist.cpp

```diff
@@ -19,12 +19,12 @@
 #include "ingameWindows/iwMusicPlayer.h"
 #include "mygettext/mygettext.h"
 #include "s25util/Log.h"
+#include "s25util/StringConversion.h"
 #include <boost/algorithm/string/trim.hpp>
 #include <boost/filesystem/path.hpp>
 #include <boost/nowide/fstream.hpp>
 #include <algorithm>
 #include <random>
-#include <sstream>
```

Comment:
This include is still required as `std::stringstream` is used. It might work without as another header could include this, but it is better to include everything you use. So please re-add this :)

Reply:
done :)